blocks/core/ff_module/ff_module-form-errors/ff_module-form-errors.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var React = require('react'),
+    getGeneratedClass = require('../../_lib/_ui/class-utils').generateStandardClass;
+
+function FormError(props) {
+    return <li className='ff_module-form-errors__message'>{props.children}</li>;
+}
+
+function FormErrors(props) {
```

Comment:
Minor: it might be useful to call this something different than the component below. Otherwise we'll have 2 different React classes called `FormErrors`.